packages/zoltan2/src/algorithms/partition/Zoltan2_AlgMultiJagged.hpp

```diff
@@ -2172,91 +2279,145 @@ mj_part_t AlgMJ<mj_scalar_t, mj_lno_t, mj_gno_t, mj_part_t>::update_part_num_arr
             //get the ideal number of parts that is close to the
             //(recursion_depth - i) root of the future_num_parts_of_part_ii.
             mj_part_t num_partitions_in_current_dim =
-                                                this->get_part_count(
-                                                                future_num_parts_of_part_ii,
-                                                                1.0 / (this->recursion_depth - current_iteration)
-                                        );
+              this->get_part_count(future_num_parts_of_part_ii,
+                                   1.0 / (this->recursion_depth - current_iteration) );
 
             if (num_partitions_in_current_dim > this->max_num_part_along_dim){
                 std::cerr << "ERROR: maxPartNo calculation is wrong. num_partitions_in_current_dim: "
-                          << num_partitions_in_current_dim <<  "this->max_num_part_along_dim:"
+                          << num_partitions_in_current_dim <<  " this->max_num_part_along_dim: "
                           << this->max_num_part_along_dim <<
-                          " this->recursion_depth:" << this->recursion_depth <<
-                          " current_iteration:" << current_iteration <<
-                          " future_num_parts_of_part_ii:" << future_num_parts_of_part_ii <<
-                          " might need to fix max part no calculation for largest_prime_first partitioning" <<
+                          " this->recursion_depth: " << this->recursion_depth <<
+                          " current_iteration: " << current_iteration <<
+                          " future_num_parts_of_part_ii: " << future_num_parts_of_part_ii <<
+                          " might need to fix max part no calculation for largest_prime_first partitioning." <<
                           std::endl;
                 exit(1);
```

Comment:
Prefer `MPI_Abort`, `std::terminate`, or throwing an exception to `exit`.

Reply:
To avoid many more merge conflicts with #5332, we are trying to touch as little code as possible in Zoltan2_AlgMultiJagged.hpp.  So for now, we'll leave this code as it was written by @mndevec .  Thanks @mhoemmen

Reply:
Thanks!